src/caselawclient/models/documents.py

```diff
@@ -191,7 +191,7 @@ def versions(self) -> list[VersionsDict]:
             return []
 
     @cached_property
-    def content_as_xml(self) -> str:
+    def content_as_xml(self) -> bytes:
```

Comment:
This might impact public-ui and editor-ui since it's a used public method.

Reply:
Hey @dragon-dxw - I've left content_as_xml as is and added a content_as_xml_bytestring method - how does that look?